src/renderer/vt/vtrenderer.hpp

```diff
@@ -15,13 +15,13 @@ Author(s):
 
 #pragma once
 
+#include <../host/VtIo.hpp>

```

Comment:
I know you've explained that these don't matter to this specific compiler, but I do think we should follow the prevailing style!

Reply:
That is: `""`.

Reply:
Actually this is even worse! This file is `src/renderer/vt/vtrenderer.hpp` and I'm trying to include `src/host/VtIo.hpp`.

With "" this would then be a MS specific search path, because MSVC is the only compiler which allows "" includes to search in the current directories of previously included files. I've replaced it with `"../../host/VtIo.hpp"`.